pytest_doctestplus/plugin.py

```diff
@@ -355,7 +359,8 @@ def parse(self, s, name=None):
                     required = []
                     skip_next = False
                     lines = entry.strip().splitlines()
-                    if any([re.match('{} doctest-skip-all'.format(comment_char), x.strip()) for x in lines]):
+                    if any([re.match(
+                            '{} doctest-skip-all'.format(comment_char), x.strip()) for x in lines]):
```

Comment:
Should work without the list and avoids running the regex on all lines if the comment was found:
```suggestion
                    if any(re.match(
                            '{} doctest-skip-all'.format(comment_char), x.strip()) for x in lines):
```